packages/core/src/modules/proofs/protocol/v1/__tests__/indy-proof-proposal.test.ts

```diff
@@ -38,9 +39,9 @@ describe('Present Proof', () => {
   test(`Alice Creates and sends Proof Proposal to Faber`, async () => {
     testLogger.test('Alice sends proof proposal to Faber')
 
-    const proposeOptions: ProposeProofOptions = {
+    const proposeOptions: ProposeProofOptions<[IndyProofFormat], [V1ProofService]> = {
```

Comment:
If you just pass this directly to `proposeProof`, you don't need to manually define the (kinda complex) types.